requirements_test_all.txt

```diff
@@ -108,10 +108,12 @@ enocean==0.50
 # homeassistant.components.season
 ephem==3.7.6.0
 
-# homeassistant.components.evohome
 # homeassistant.components.honeywell
 evohomeclient==0.3.2
 
+# homeassistant.components.evohome
+evohomeclient==0.3.3
```

Comment:
This won't work without also upgrading the version in the honeywell integration.

Reply:
woops - so busy with the detail, forgot the big things: thanks.  Added myself as codeowner too.